src/ui/button/styles/color.js

```diff
@@ -49,4 +51,17 @@ export const buttonColorStyle = `
         box-shadow: 0px 0px 1px 3px #0c67ff inset;
     }
 
+    .${ CLASS.BUTTON }.${ CLASS.COLOR }-${ BUTTON_COLOR.SILVER } {
+        background: #eeeeee;
+        color: #111;
+    }
+    .${ CLASS.BUTTON }.${ CLASS.COLOR }-${ BUTTON_COLOR.SILVER }:hover {
+        filter: brightness(1.9);
+        background: #00000020;
```

Comment:
I don't think this is a valid color code. They should be 6 digits. Did you mean to use `#000020`?